Installation/cmake/modules/CGALConfig_install.cmake.in

```diff
@@ -85,6 +86,53 @@ macro(check_cgal_component COMPONENT)
     if ( WITH_${CGAL_LIB} )
       if(TARGET CGAL::${CGAL_LIB} OR CGAL_HEADER_ONLY )
         set( ${CGAL_LIB}_FOUND TRUE )
+        if ("${CGAL_LIB}" STREQUAL "CGAL_Qt5")
+          find_package(OpenGL QUIET)
+          find_package(Qt5 COMPONENTS OpenGL Svg QUIET)
+
+          set(CGAL_Qt5_MISSING_DEPS "")
+          if(NOT Qt5OpenGL_FOUND)
+            set(CGAL_Qt5_MISSING_DEPS "Qt5OpenGL")
+          endif()
+          if(NOT Qt5Svg_FOUND)
+            set(CGAL_Qt5_MISSING_DEPS "${CGAL_Qt5_MISSING_DEPS} Qt5Svg")
+          endif()
+          if(NOT Qt5_FOUND)
+            set(CGAL_Qt5_MISSING_DEPS "${CGAL_Qt5_MISSING_DEPS} Qt5")
+          endif()
+          if(NOT OPENGL_FOUND)
+            set(CGAL_Qt5_MISSING_DEPS "${CGAL_Qt5_MISSING_DEPS} OpenGL")
+          endif()
+
+          if(CGAL_Qt5_MISSING_DEPS)
+            set( ${CGAL_LIB}_FOUND FALSE )
+            message(STATUS "Missing the dependencies: ${CGAL_Qt5_MISSING_DEPS} cannot be configured.")
+          else()
+            set( ${CGAL_LIB}_FOUND TRUE )
+            if (CGAL_HEADER_ONLY)
+              message("**************************** ${CGAL_GRAPHICSVIEW_PACKAGE_DIR}")
+              include("${CGAL_MODULES_DIR}/Use_CGAL_Qt5_headers.cmake")
+            endif()
+          endif()          
+        else()
+          if ("${CGAL_LIB}" STREQUAL "CGAL_ImageIO")
+            find_package( OpenGL QUIET )
+            find_package( ZLIB QUIET )
+
+            if(OPENGL_FOUND)
+              set( ${CGAL_LIB}_FOUND TRUE )
+            else(OPENGL_FOUND)
+              set( ${CGAL_LIB}_FOUND FALSE )
+              message(STATUS "Notice: ImageIO disabled because it requires OpenGL.")
+            endif(OPENGL_FOUND)
+          else()
+            # Librairies that have no dependencies
+            set( ${CGAL_LIB}_FOUND TRUE )
+          endif()
+        else()
+          # Librairies that have no dependencies
+          set( ${CGAL_LIB}_FOUND TRUE )
+        endif()
```

Comment:
The macro `check_cgal_component` is very similar in `CGALConfig_install.cmake.in` and in `CGALConfig_binary.cmake.in`. It should be in a new dedicated `.cmake` file that is included by both files.